frontend/src/util/AAC/Speech.ts

```diff
@@ -0,0 +1,10 @@
+export function speakViaWebSpeechAPI(sound: string) {
+    if (!("speechSynthesis" in window)) {
```

Comment:
Where does "speechSynthesis" get added to the window? If it doesn't exist, shouldn't we create it?

Reply:
speechSynthesis is not present in the NextJs runtime its only possible on the browser (window) so we have to prevent nextjs from accessing it.

Reply:
Why is speechSynthesis not present in the NextJs runtime? My understanding is that during "runtime" the PWA running in a bare-bones browser window.

Reply:
NextJs compiles pages on the server, the server does not access to browser level APIs like the window object which contains speechSynthesis. When the code is shipped to the browser the object (window) can be accessed. The server runtime has access to things the browser does not have like ENV secrets or file system access.